src/index.md

```diff
@@ -103,7 +103,8 @@ Jul 10, 2021 | [MFEM Community Workshop](workshop.md) in October.
 **Download mfem-4.3.tgz**
 </button>](https://bit.ly/mfem-4-3)
 
-[Older releases](download.md) ┊ [Python wrapper](https://github.com/mfem/PyMFEM)
+[Older releases](download.md) ┊ [Python wrapper](https://github.com/mfem/PyMFEM) ┊ [Jupyter notebook](https://mybinder.org/v2/gh/mfem/mfem/master?filepath=examples%2Fjupyter%2Fex.ipynb)
+
```

Comment:
Some alternatives to consider:
1. Use the standard "launch binder" badge with tooltip "C++ Jupiter notebook":
```suggestion
[Older releases](download.md) ┊ [Python wrapper](https://github.com/mfem/PyMFEM) ┊
[![Launch Binder](https://mybinder.org/badge_logo.svg){style="display:inline;margin:0"}](
https://mybinder.org/v2/gh/mfem/mfem/master?filepath=examples%2Fjupyter%2Fex.ipynb
 "C++ Jupiter notebok")
```
2. Add a button next the the big "Download" button above -- similar to the "Use web version" button on the GLVis website. The button text can say something like "Try on Binder", or "C++ Jupiter notebook".

I think I like the second option a bit more.

Reply:
@tomstitt, what do you think?

Reply:
I like the one with the binder link!